test/index.test.js

```diff
@@ -2,8 +2,8 @@
  * @jest-environment jsdom
  */
 
-const axios = require('../dist/index')
```

Comment:
The idea behind to use require from builded files is avoiding to use any transpiled process such babel.

Reply:
I see your point but that would mean that I can no *unit* test modules but only the bundle as a whole. And here I really prefer to test the error handler function in isolation without going through all of axios.

So I don't think adding a transpiler to be able to use ts code in jest tests isn't a bad change.